pallets/creditcoin/src/lib.rs

```diff
@@ -1,6 +1,16 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![allow(clippy::too_many_arguments, clippy::type_complexity)]
 
+#[macro_export]
+macro_rules! warn_or_panic {
```

Comment:
Pedantic, this is more like warn_or_(warn and panic).

Reply:
From the use cases, it doesn't look like to need the tt muncher. You can use a regular function for this if you don't ever plan to have more than one logging line.

Reply:
I'm not sure I follow what you mean here.